app/src/main/java/org/mozilla/fenix/theme/FirefoxDimens.kt

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@file:Suppress("ConstructorParameterNaming")
+
+package org.mozilla.fenix.theme
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * A grid system. Smaller components can align to a 'sub' grid [grid_0_25] or [grid_0_5].
```

Comment:
I am wondering where this grid system is coming from? I am a bit hesitant to ship a grid system that isn't guided by our UX design system.

Reply:
Hmm the thinking was that we are hard coding these dimensions on the screens, and it might help to centralize them as a grid system. 

We can look into aligning them with the design system and involve UX in it. Although that could be a step ahead, we could look into how to use the already hardcoded dimensions as a common variable of the theme as a first improvement. wdyt?